uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/OpenTabActionUiTest.scala

```diff
@@ -0,0 +1,42 @@
+package com.virtuslab.gitmachete.uitest
+
+import java.io.File
+import java.net.URI
+
+import com.virtuslab.ideprobe.dependencies._
+import com.virtuslab.ideprobe.ide.intellij.{ DriverConfig, IntelliJFactory }
+import com.virtuslab.ideprobe.protocol.IdeMessage.Level.Info
+import com.virtuslab.ideprobe.{ IntegrationTestSuite, IntelliJFixture, WorkspaceConfig, WorkspaceTemplate }
+import org.junit.{ Assert, Test }
+
+class OpenTabActionUiTest extends IntegrationTestSuite {
```

Comment:
Just too much hassle writing that in Java (using code compiled in Java from Scala is trivial; the reverse is possible but generally PITA)